src/PerfView/SupportFiles/HtmlReportUsersGuide.htm

```diff
@@ -120,30 +160,56 @@ <h4>Expected Win from Background JIT Compilation</h4>
     <ol>
-        <li>For that process, a set of top level statistics are displayed indicating how many JIT compilations happened in the background.&nbsp; The most important of these metrics is the &#39;<strong>JIT time NOT moved to background thread</strong>&#39;.&nbsp; This is the JIT time that matters because it is left on the foreground thread where it slows down the application.&nbsp; The difference between this number and the JIT time on a run without background JIT is the &#39;win&#39; of performing this optimization.&nbsp; </li>
+        <li>Each process has a set of top level statistics indicating how many JIT compilations happened in the foreground and background.&nbsp; Any JIT time that remains on the foreground thread slows down the application startup.&nbsp; The difference between the foreground times on runs with and without background JIT is the &#39;win&#39; of performing this optimization.&nbsp; </li>
         <li>There is a hyperlink to the a CSV spreadsheet displaying detailed diagnostics of the what was JIT compiled in the background as well as what was recorded for the next launch of the program.&nbsp; </li>
-        <li>The &#39;BG&#39; column of each JIT compiled method is set to indicate whether the JIT compilation happened in the background or not.</li>
+        <li>The &#39;Trigger&#39; column of each method indicates whether the compilation happened in the foreground.</li>
     </ol>
     <p><strong>What can go wrong with background JIT compilation.</strong></p>
-        <p>It your program has used SetProfileRoot and StartProfile, but JIT compilation (as shown in the JITStats view) does not show any or very little background JIT compilation, there are several issues that may be responsible.&nbsp;&nbsp; Fundamentally a important design goal was to insure that background JIT compilation did not change the behavior of the program under <strong>any circumstances</strong>.&nbsp;&nbsp;&nbsp; Unfortunately, it means that the algorithm tends to bail out quickly.&nbsp;&nbsp; In particular</p>
+    <p>It your program has used SetProfileRoot and StartProfile, but JIT compilation (as shown in the JITStats view) does not show any or very little background JIT compilation, there are several issues that may be responsible.&nbsp;&nbsp; Fundamentally a important design goal was to insure that background JIT compilation did not change the behavior of the program under <strong>any circumstances</strong>.&nbsp;&nbsp;&nbsp; Unfortunately, it means that the algorithm tends to bail out quickly.&nbsp;&nbsp; In particular</p>
```

Comment:
I realize this is not your error, but Can you change 

It your program -> If your program

Reply:
Np